amazon-msk/cdktf/iot-ingest-and-control/zilla.mustache

```diff
@@ -0,0 +1,133 @@
+name: {{name}}
+vaults:
+  secure:
+    type: aws-secrets
+{{^cloudwatchDisabled}}
+telemetry:
+  metrics:
+    - stream.active.received
+    - stream.active.sent
+    - stream.opens.received
+    - stream.opens.sent
+    - stream.data.received
+    - stream.data.sent
+    - stream.errors.received
+    - stream.errors.sent
+    - stream.closes.received
+    - stream.closes.sent
+  exporters:
+    stdout_logs_exporter:
+      type: stdout
+    aws0:
+      type: aws-cloudwatch
+      options:
+        logs:
+          group: {{cloudWatchLogsGroup}}
+          stream: events
+        metrics:
+          namespace: {{cloudWatchMetricsNamespace}}
```

Comment:
Does it make sense to tighten the `cloudwatch` checks around the `aws-cloudwatch` exporter, so we have metrics recorded locally even if not exported to AWS CloudWatch?